lib/sys/sys.go

```diff
@@ -229,3 +231,48 @@ func FilePermission() os.FileMode {
 	//return 0644
 	return 0755
 }
+
+func Serialize(v interface{}, filename string) {
```

Comment:
@guoci  Is this new method based on v5? or it is something that you created?

Reply:
This function requires `github.com/vmihailenco/msgpack/v5`. I wrote that function.